modules/rosa-required-aws-service-quotas.adoc

```diff
@@ -30,35 +38,57 @@ If you need to modify or increase a specific quota, see Amazon's documentation o
 |ec2
 |L-1216C47A
 |5
-|100
-| Maximum number of vCPUs assigned to the Running On-Demand Standard (A, C, D, H, I, M, R, T, Z) instances.
-
-The default value of 5 vCPUs is not sufficient to create ROSA clusters. ROSA has a minimum requirement of 100 vCPUs for cluster creation.
-
+a|
+ifndef::openshift-rosa-hcp[]
+100
+endif::[]
+ifdef::openshift-rosa-hcp[]
+32
+endif::[]
+|Maximum number of vCPUs assigned to the Running On-Demand Standard (A, C, D, H, I, M, R, T, Z) instances. The default value of 5 vCPUs is not sufficient to create ROSA clusters.
```

Comment:
Should "Running On-Demand...." be in back ticks like here https://github.com/openshift/openshift-docs/pull/82688/files#diff-3194eeff671f12019cc41e511d72071149c1c1fa1e474d9302a13cfcf9690675R18